src/vivarium_gates_iv_iron/components/hemoglobin.py

```diff
@@ -156,6 +176,16 @@ def anemia_source(self, index: pd.Index) -> pd.Series:
             name="anemia_levels",
         )
 
+    def maternal_disorder_risk_effect(self, index: pd.Index) -> pd.Series:
+        hemoglobin_level = self.hemoglobin(index)
+        rr = self.maternal_disorder_rr(index)
+        paf = self.maternal_disorder_paf(index)["value"]
+        tmrel = TMREL_HEMOGLOBIN_ON_MATERNAL_DISORDERS
+        per_simulant_exposure = (tmrel - hemoglobin_level + abs(tmrel - hemoglobin_level)) / 2 / RR_SCALAR
```

Comment:
This is 0 if `hemoglobin >= tmrel` ? If that's the intent, wouldn't it be clearer to do that explicitly rather that using `abs`? You wouldn't need to divide by 2 either then.

Reply:
That's the intent. Since it's a one-time operation in the artifact, adding inefficiency for legibility probably would be a good trade-off. Here, I'm following the specification: https://vivarium-research.readthedocs.io/en/latest/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_effects/index.html#id18